Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM python:2.7-slim
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+		dos2unix \
+		gcc \
+		libc-dev \
+	&& rm -rf /var/lib/apt/lists/*
+
+ENV TINI_VERSION v0.13.2
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
+RUN chmod +x /tini
+
+ENV RQ_VERSION 0.7.1
+RUN pip install rq==$RQ_VERSION
+
+ADD autoexec.sh /
+RUN dos2unix /autoexec.sh
```

Comment:
Why do we have CRLF line endings here? Maybe It will be better to fix file encoding at first place?